LayoutTests/media/now-playing-webaudio.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Testing basic video exchange from offerer to receiver</title>
+        <script src="../resources/testharness.js"></script>
+        <script src="../resources/testharnessreport.js"></script>
+    </head>
+    <body>
+        <script>
+function waitFor(delay)
+{
+    return new Promise(resolve => setTimeout(resolve, delay));
+}
+
+async function waitForCriteria(test, criteria)
+{
+    let counter = 0;
+    while (!criteria() && ++counter < 100)
+        await waitFor(50);
+}
+
+promise_test(async test => {
+    if (!window.internals)
+        return;
+
+    navigator.audioSession.type = "playback";
+
+    let context = new AudioContext();
+
+    waitFor(100);
```

Comment:
no await ? otherwise this serves no purpose

Reply:
Right, will fix (or remove).